py/server/deephaven/updateby_clause.py

```diff
@@ -0,0 +1,269 @@
+#
+#     Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+"""This module supports building various update-by clauses for use with the update-by operations on tables. """
+from enum import Enum
+from typing import Union, List
+
+import jpy
+
+from deephaven import DHError
+from deephaven._wrapper import JObjectWrapper
+from deephaven.jcompat import to_sequence
+
+_JUpdateByClause = jpy.get_type("io.deephaven.api.updateby.UpdateByClause")
+_JBadDataBehavior = jpy.get_type("io.deephaven.api.updateby.BadDataBehavior")
+_JEmaControl = jpy.get_type("io.deephaven.api.updateby.EmaControl")
+_JMathContext = jpy.get_type("java.math.MathContext")
+
+
+class MathContext(Enum):
```

Comment:
1) No docstring
2) I see that this is mirroring a Java class named "MathContext".  The word "Context" is heavily overloaded in DH.  Another name would likely be more clear for a user.